.github/ISSUE_TEMPLATES/feature.md

```diff
@@ -0,0 +1,21 @@
+---
+
+name: 🚀 Feature Proposal  
+about: Create a report to help us improve
+---
+
+## 🚀 Feature Proposal
+
+A clear and concise description of what the feature is.
+
+## Motivation
+
+Please outline the motivation for the proposal.
+
+## Example
+
+Please provide an example for how this feature would be used.
+
+## Pitch
+
+Why does this feature belong in Jest core?
```

Comment:
Maybe include some text around matchers (that we rarely accept them), etc?

Reply:
+1, I also added a couple more common examples